docs: cleanup all sections for consistency (#186)
Open-Attestation/documentation-website: website/docs/docs-section/appendix/glossary.md

An OA document means an OpenAttestation document, which is issued using the OpenAttestation framework. The data in the OA document is stored in a `.json` file. It is a tamper-evident document that cryptographically proves the issuer.

> There are two categories of OA document. One type refers to the [verifiable document](#verifiable-document), while the other type refers to the [transferable record](#transferable-record).

Reviewer: ```suggestion
There are two types of OA documents. One type being [verifiable documents](#verifiable-document), while the other being [transferable records](#transferable-record).
```

Author: updated with minor adjustment in the punctuation
